scripts/metricbeat-paths

```diff
@@ -0,0 +1,24 @@
+#! /usr/bin/env bash
+
+if [[ $(uname) == "Linux" ]]
+then
+  echo "Host is Linux, enabling /proc and /sys/fs/cgroup Metricbeat monitoring"
+  if [[ $(grep METRICBEAT_PROC_PATH .env -c) == "0" ]]
+  then
+    echo METRICBEAT_PROC_PATH=/proc >> .env
+  fi
+  if [[ $(grep METRICBEAT_CGROUP_PATH .env -c) == "0" ]]
+  then
+    echo METRICBEAT_CGROUP_PATH=/sys/fs/cgroup >> .env
+  fi
+else
+  echo "Host is not Linux, disabling /proc and /sys/fs/cgroup Metricbeat monitoring"
+  if [[ $(grep METRICBEAT_PROC_PATH .env -c) != "0" ]]
+  then
+    sed -i '/METRICBEAT_.*_PATH/d' .env
```

Comment:
Output on macos

```
=== Checking Host OS...
Host is not Linux, disabling /proc and /sys/fs/cgroup Metricbeat monitoring
sed: 1: ".env": invalid command code .
```

Reply:
Thanks, this one should resolve that https://github.com/place-labs/partner-environment/pull/34/commits/e5b390a78568d724287b15e20f4276b3384be8c8